LibertyDSNP/sdk-ts
# Features/GitHub pages 177624525

Problem
=======
Put the docs in GitHub Pages and remove the markdown from being committed.
[link to Pivotal Tracker #177624525](https://www.pivotaltracker.com/story/show/177624525)

Solution
========
Copied most of the setup from the spec repo

Change summary:
---------------
*  Cleanup markdown docs
* Upgrade lock file and typedoc versions
* GitHub Actions deploy docs 

Steps to Verify:
----------------
1. Visit the github pages for this repo and see the docs at https://libertydsnp.github.io/sdk-ts/
